docs/api_clients.md

```diff
@@ -0,0 +1,197 @@
+# Api clients
+
+Consuming webservices is a recurrent job on server-side development these days.
+Et over and over again, we implement our client, using Guzzle (or other library), everytime better, but everytime different.
+
+Api client component helps implementation of your http webservices calls : it give you some classes to exploit Guzzle, Symfony Routing and Majora Agnostic router, in a query builder Api style.
```

Comment:
Api client component helps implementation of your http webservices calls ->  
Api client component helps implementation of your calls to http webservices